### ISSUE TITLE
Immediate disconnect?

### ISSUE THREAD

**Issue by LuckyURE** [3ventic/EvtSource#4]
Hello,

Thank you for creating this, I couldn't find any other way to connect to an SSE service using c# on .Net Core 2.0!  My code looks like this:

```
var evt = new EventSourceReader(new Uri(url)).Start();
            evt.MessageReceived += (sender, response) => Console.WriteLine($"{response.Event} : {response.Message}");
            evt.Disconnected += async (sender, response) => {
                Console.WriteLine($"Retry: {response.ReconnectDelay} - Error: {response.Exception}");
                await Task.Delay(response.ReconnectDelay);
                evt.Start(); // Reconnect to the same URL
            };
```
The URL I'm connecting to is: https://horizon-testnet.stellar.org/accounts/GDPQINE6WPAHGZBG2R5KLH7DXGNCLBXTLIYMF53QH4QA36OOZY2TF2O6/payments

Which as you can see returns JSON data just fine.  I know that if I'm using curl I have to pass the Accept: text/event-stream header which I'm assuming your code is doing.

But I keep getting a message saying Retry: 3000 - Error: "it's blank, nothing is reported"

Any ideas?

**Comment by 3ventic** [comment]
Do you mean there's nothing after `Error:` or that it reads `"it's blank, nothing is reported"`?

**Comment by LuckyURE** [comment]
There is nothing after error.  

That url is public, feel free to try it and replicate the issue.

Thanks for looking into this.

**Comment by LuckyURE** [comment]
Hey, wanted to let you know I fixed the issue today.  I pulled down your repository and modified it to work on .Net 4.5.2 (we aren't allowed to have dotnet core on our machines yet).

I manually added the header headers.Add("Accept", "text/event-stream"); to the HttpClient that you created and everything started working great.  Getting timeout errors every so often but it keeps chugging along anyway like it should.

Not sure if you or others need to add that header (or least provide the option to add it in the library perhaps to those that need it, but that's what worked for me.  Thanks for your help and hopefully this helps others!
